ge/org.osate.ge/src/org/osate/ge/internal/util/ContentFilterUtil.java

```diff
@@ -45,12 +45,7 @@ private ContentFilterUtil() {
 	 * @return true if the specified business object passes any of the specified content filters
 	 */
 	public static boolean passesAnyContentFilter(final Object bo, final ImmutableSet<ContentFilter> contentFilters) {
-		for (final ContentFilter filter : contentFilters) {
-			if (filter.test(bo)) {
-				return true;
-			}
-		}
-		return false;
+		return contentFilters.stream().map(filter -> filter.test(bo)).anyMatch(Boolean::valueOf);
```

Comment:
You can do an "anyMatch" with the filter directly instead of a map and then "anyMatch"